pkg/deploy/lattice/access_log_subscription_manager.go

```diff
@@ -59,31 +61,83 @@ func (m *defaultAccessLogSubscriptionManager) Create(
 		return nil, fmt.Errorf("unsupported source type: %s", accessLogSubscription.Spec.SourceType)
 	}
 
+	tags := m.cloud.DefaultTags()
```

Comment:
I'm think we can benefit from new method in cloud that will take an argument of tags and merge with default tags. I can see more places that will need to define additional tags together with default ones, would be nice just to pass map of values.
for example:

```go
func (c *cloud) DefaultTagsMerge(tags Tags) Tags {
  return maps.Copy(c.DefalutTags(), tags)
}

m.cloud.DefaultTagsMerge({lattice.AccessLogPolicyTagKey}: aws.String(accessLogSubscription.Spec.ALPNamespacedName.String())})
```

Reply:
Love this idea, will do next commit!